src/background.ts

```diff
@@ -326,10 +328,45 @@ browser.runtime.onInstalled.addListener(async (details) => {
           }
         }
       }
-      if (convertVersionToNumber(details.previousVersion) < 300) {
-        store.dispatch({
-          type: ReduxConstants.RESET_COOKIE_DELETED_COUNTER,
-        });
+      // Language Migration 3.7.0+
+      if (convertVersionToNumber(details.previousVersion) < 370) {
```

Comment:
I recommend adding a test for this if you haven't already (hard to tell in this large PR), just to verify the behavior from one state to another. Another option is to split the setting migration + test into a separate PR.